docs/Privacy/Tutorials/Configuring-Privacy.md

```diff
@@ -3,6 +3,10 @@ description: Configuring Privacy
 
 # Configuring a Network for Private Transactions 
 
+### Prerequisites
+
+* [Orion](https://docs.orion.pegasys.tech/en/latest/Installation/Build-From-Source/)
```

Comment:
Do we really have to build Orion from sources for this?

Reply:
No we don't. We can currently only link to the "Install Binary Distribution", "Build from Source", or "Dependencies" topics because the documentation lacks an "Installation Overview" topic. I therefore raised a seperate Jira issue to add that content. I'll assign that Jira issue to myself and link to that new topic when it is published.

Reply:
I've updated the link to point to the Installation Overview topic.